### PR TITLE
Patch the .desktop file to add StartupWMClass

### PR DESCRIPTION
**Pull request by stan-janssen**
This prevents the Dock icon from being duplicated on elemenatryOS

Please see https://github.com/elementary/dock/issues/64 for more information

### CODE REVIEW

**Comment by gasinvein** [comment]
Did you sent this patch upstream? If not, please do so.

File: org.fritzing.Fritzing.json [hunk 1]
```diff
@@ -72,7 +72,8 @@
                 {
                     "type": "patch",
                     "paths": [
-                        "patches/fritzing-appdata-screenshots.patch"
+                        "patches/fritzing-appdata-screenshots.patch",
+                        "patcheds/add-wm-class.patch"
```
Comment (gasinvein) [inline org.fritzing.Fritzing.json]: A typo?
```suggestion
                        "patches/add-wm-class.patch"
```

Comment (stan-janssen) [reply org.fritzing.Fritzing.json]: Whoops, yes that was a typo. I just fixed it in my branch.

**Comment by stan-janssen** [comment]
The problem only occurs when running the application as a flatpak application, so I thought is was more approprate to add it here.

**Comment by gasinvein** [comment]
This is a sensible change regardless of flatpak - declaring WM class explicitly won't hurt.

**Comment by stan-janssen** [comment]
I also offered the fix upstream, let's wait for their response.

**Comment by gasinvein** [comment]
bot, build

**Comment by gasinvein** [comment]
@stan-janssen Can you please install the test build and verify that it fixes the problem?

**Comment by stan-janssen** [comment]
I just installed it and it indeed fixes the problem.

As mentioned above, I also contributed this fix to the upstream fritzing-app repository.

**Comment by gasinvein** [comment]
Thanks!

**Comment by stan-janssen** [comment]
Just to be clear: if this same fix gets accepted upstream, we would have to remove the patch from this flatpak version again, correct?

**Comment by gasinvein** [comment]
Once upstream releases a version that includes the fix, yes.

**Pull request merged**